functions/create_draft/interactivity_handler.ts

```diff
@@ -23,7 +23,8 @@ import DraftDatastore from "../../datastores/drafts.ts";
 
 export const openDraftEditView: BlockActionHandler<
   typeof CreateDraftFunction.definition
-> = async ({ body, action, client }) => {
+> = async ({ body, action, client, inputs }) => {
+  // If the user selects to edit the draft message
   if (action.selected_option.value == "edit_message_overflow") {
```

Comment:
Not a blocker at all, but might be nice to change these to `===` 👀

Reply:
For sure!